fxa-rust-client/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "fxa-client"
+version = "0.1.0"
+authors = ["Edouard Oger <eoger@fastmail.com>"]
+
+[dependencies]
+base64 = "0.9.0"
+jose = { git = "https://github.com/eoger/rust-jose" }
+lazy_static = "1.0.0"
+log = "0.4"
+serde = "1.0"
+serde_derive = "1.0"
+serde_json = "1.0"
+url = "1.6.0"
+rand = "0.5.0-pre.1"
+regex = "1.0.0"
+reqwest = "0.8.2"
+hex = "0.3.1"
+hkdf = "0.4"
+hmac = "0.6.2"
+sha2 = "0.7.0"
+hawk = { git = "https://github.com/eoger/rust-hawk", branch = "use-openssl" }
```

Comment:
Upstream is at https://github.com/taskcluster/rust-hawk/ but not maintained, I believe we should contact them and take ownership of the repo since we'll be using hawk extensively.